spec/haml_lint/configuration_loader_spec.rb

```diff
@@ -109,6 +109,146 @@
         expect { subject }.to raise_error HamlLint::Exceptions::ConfigurationError
       end
     end
+
+    context 'with an "inherits_from" directive' do
+      let(:config_file) do
+        [
+          "inherits_from: #{other_config}",
+          'skip_frontmatter: true',
+          'linters:',
+          '  AltText:',
+          '    enabled: true'
+        ].join("\n")
+      end
+      let(:other_config) { 'other_haml-lint.yml' }
+
+      before do
+        File.open(file_name, 'w') { |f| f.write(config_file) }
```

Comment:
Ah, `file_name` wasn't added by you. And I take it the `around` block up top takes care of cleaning things up.